tools/cwrap/plugins/THPPlugin.py

```diff
@@ -191,8 +197,8 @@ def _allocate(typename, tmpl, cuda_tmpl=None, sparse=False):
     }
 
     OUT_INIT = """
-    __out = kwargs ? PyDict_GetItemString(kwargs, "out") : NULL;
-    if (__out == Py_None) { __out = NULL; __dictcount--; __argcount--; }
+    ___out = kwargs ? PyDict_GetItemString(kwargs, "out") : NULL;
+    if (___out == Py_None) { ___out = NULL; __dictcount--; __argcount--; }
```

Comment:
what's the deal with an extra underscore?

Reply:
__out is already defined in MSVC as sth else.